style.css

```diff
@@ -55,3 +55,21 @@ section p {
   line-height: 1.5;
   max-width: 55ch;
 }
+
```

Comment:
Do you want to talk through what you were going for here?

(In general don't leave dead/commented out code in your PRs)